docker/defaults.yaml

```diff
@@ -2,9 +2,9 @@
 
 docker:
   process_signature: '/usr/bin/docker'
-  install_docker_py: True
-  pip_version: '8.1.1'
```

Comment:
^^ that was wrong in more than one way. It wanted the pip version of the docker-py package to install.